js/list.controller.js

```diff
@@ -0,0 +1,16 @@
+angular.module('imdbControllers', [])
+
+angular.module('imdbControllers')
+.controller('ListController', function($scope, IMDBFactory) {
+	IMDBFactory.getMovies().success(function(data) {
```

Comment:
I know we didnt cover the error callback but you can include an error callback in addition to the success callback to handle when a call the REST API fails.